src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const rows = [...table.tBodies[0].rows];
+
+table.addEventListener('click', event => {
```

Comment:
put an event listener just on the table header, now I can click on the table rows itself and the sorting would apply also